Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -53,6 +53,7 @@
     private TagEditText mTagEditText;    /* Chips view editor for tags */
 
     private int         mBackgroundColor;
+    private boolean     mNoteChanged;
```

Comment:
I removed `mNoteChanged`, it is unnecessary. In `saveNote()` method it is checked that if `mNote` is changed ( `final boolean anythingChanged` ).  But it saves the note if `mAutoSaveEnabled` is `true`. I will add a `forceSave` parameter to the `save()` method it will save the note without enabling auto-save. Is this ok?